apps/server/lib/lexical/server/code_intelligence/completion.ex

```diff
@@ -107,30 +103,7 @@ defmodule Lexical.Server.CodeIntelligence.Completion do
   end
 
   defp context_will_give_meaningful_completions?(%Env{} = env) do
-    case Code.Fragment.cursor_context(env.prefix) do
-      {:local_or_var, name} ->
-        local_length = length(name)
-
-        surround_begin =
-          Math.clamp(env.position.character - local_length - 1, 1, env.position.character)
-
-        case Code.Fragment.surround_context(env.prefix, {1, surround_begin}) do
-          :none ->
-            local_length > 1
-
-          _other ->
-            true
-        end
-
-      :none ->
-        false
-
-      {:unquoted_atom, name} ->
-        length(name) > 1
-
-      _ ->
-        true
-    end
+    Code.Fragment.cursor_context(env.prefix) != :none
```

Comment:
this change spooks me a little

Reply:
Unless I'm reading the original code wrong, it was basically just ensuring that the cursor context wasn't `:none` and the prefix, if it was a `:local_or_var` or an `:atom`, had a length greater than 1.